src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/response/model/SentenceAndOffenceDetails.kt

```diff
@@ -1,22 +1,70 @@
 package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.response.model
 
+import jakarta.validation.constraints.NotNull
+import java.time.LocalDate
+
 class SentenceAndOffenceDetails {
```

Comment:
These are huge classes. I was under the impression that we only needed a specific few fields from each external API. If we don't need all these fields then we should reduce the size of the projection DTO that we're defining.

Reply:
Again something to check I guess. I've taken the entire response from the prison-api. Couldn't find in the trello ticket on which fields were required and which weren't

Reply:
This is sorted now, we're only returning one field